### ISSUE TITLE
Accessibility with VoiceOver: cannot select buttons and text in custom MessageView

### ISSUE THREAD

**Issue by droidery** [SwiftKickMobile/SwiftMessages#78]
We've received a bug report from one of our blind users unable to use a MessageView in our iOS app and I noticed it was due to the recent VoiceOver support changes in v 3.3.0.

We use a custom MessageView like the Taco MessageView, with buttons, titles, sliders etc. With the new VoiceOver support you can no longer "browse" the custom MessageView using VoiceOver by moving your finger around the screen.

Instead, only the title and description is read ("Just one please, how many tacos would you like?"). The expected behavior is that a blind user can:
1. read title and subtitle by moving finger over text (currently VoiceOver reads title + subtitle at once)
2. read instructions (currently VoiceOver skips it) 
2. interact with buttons in MessageView by moving finger over button (SwiftMessages blocks interaction)
3. can use slider
4. can dismiss MessageView by tapping outside frame of MessageView (currently supported 👍 )

As an example, the user can't tap the "get tacos" button because VoiceOver only reads out "Just one please, how many tacos would you like?" again.

It would be nice to be able to disable the new VoiceOver support in v 3.3.0 as it makes things very difficult for custom MessageView's with blind users.

<img width="1019" alt="screenshot" src="https://cloud.githubusercontent.com/assets/25188027/24289734/a1f5c942-10d6-11e7-8e35-821d9b06d024.png">

**Comment by wtmoose** [comment]
Could you take a look at the `AccessibleMessage` protocol?

**Comment by droidery** [comment]
@wtmoose  I've taken a look at `AccessibleMessage`. I could be misunderstanding the protocol, but it wouldn't be useful for custom message views (like the demo one) which needs more accessibility elements than the protocol allows?

Try enabling VoiceOver on your phone, and opening the demo app with the Taco view. The VoiceOver blocks user interaction with the message, and you can't read title/subtitle/description by moving your finger to each element.

**Comment by wtmoose** [comment]
You can return as many `additonalAccessibilityElements` in your custom class.

I'm about to post a better default implementation of `additonalAccessibilityElements` that I think will cover your use case automatically.

**Comment by wtmoose** [comment]
Can you check out the head of master and let me know how it works for you?

Here are the release notes:

* `MessageView` is smarter about including additional accessibility views for cases where you've added accessible elements to the view. Previously only the `button` was included. Now all views where `isAccessibilityElement == true` are included.

    Note that all nib files now have `isAccessibilityElement == false` for `titleLabel`, `bodyLabel` and `iconLabel` (`titleLabel` and `bodyLabel` are read out as part of the overall message view's text). If any of these need to be directly accessible, then copy the nib file into your project and select "Enabled" in the Accessibility section of the Identity Inspector.

**Comment by droidery** [comment]
Thanks @wtmoose, that's really good.

Is this for the next version?

**Comment by wtmoose** [comment]
Cool. I'll release it this weekend or shortly thereafter

**Comment by wtmoose** [comment]
Published 3.3.2 with this fix.
